README.md

```diff
@@ -82,9 +82,8 @@ About
 Curry is maintained and funded by thoughtbot, inc. The names and logos for
 thoughtbot are trademarks of thoughtbot, inc.
 
-We love open source software! See [our other projects][community] or look at
-our product [case studies] and [hire us][hire] to help build your iOS app.
+We love open source software! See [our other projects](https://thoughtbot.com/tools) or [see some case studies][case studies]. We also have [iOS learning resources at Upcase by thoughtbot](https://thoughtbot.com/upcase/ios).
 
 [community]: https://thoughtbot.com/community?utm_source=github
-[case studies]: https://thoughtbot.com/ios?utm_source=github
+[case studies]: https://thoughtbot.com/ios
```

Comment:
The `utm_source` on thoughtbot.com is often captured in the session and if the user submits a form, we store the source in the CRM, so can run reports on the last touch all the way through the end of the sales pipeline for projects that close. Not a big deal to remove it if you'd like, but maybe we use the param consistently one way or the other on these links?